script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2491,7 +2496,7 @@ After all this time――there's no use going back to the Shiki that came before
 [sha:b375c4198b79ecf2cf01e96c47d4153ccc1a97fb]{
 -- Page 149, Offset 42410.
 -- 「ああ、そうそう。彼女のことだけど、僕も気に入ったよ。
-"Oh, one last thing――I liked her too.
+"Oh, one last thing――about your girlfriend... I liked her too.
```

Comment:
Kinda JP phrasing. I'd integrate it more, or leave it dropped and assume the reader possesses the wherewithal to work it out.
> Oh, and about your girlfriend... I liked her too.
> Oh, and one last thing... I liked your girlfriend too.